riotfile.py

```diff
@@ -607,16 +608,13 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="flask",
             command="pytest {cmdargs} tests/contrib/flask",
-            pkgs={
-                "blinker": latest,
-            },
+            pkgs={"blinker": latest,"pytest": "~=3.0", "more_itertools": "<8.11.0"},
```

Comment:
I don't think we want to be pinning pytest to 3 for all versions of flask. pytest 3 is 3 years old to the day
<img width="315" alt="image" src="https://user-images.githubusercontent.com/6321485/141344721-0dc17afa-888c-4c3f-bb4b-afd0bdb4e98b.png">

Why was this change necessary?